pyproject.toml

```diff
@@ -1,12 +1,16 @@
 [tool.poetry]
 name = "openhands"
-version = "0.8.3"
+version = "0.8.4"
```

Comment:
We probably shouldn't cut a new release until we're pretty sure that we don't have any regressions in agent quality. I think @mamoodi is taking a final look now.